server/service/http/api.go

```diff
@@ -99,7 +109,9 @@ func (a *API) respondError(w http.ResponseWriter, apiErr coderr.CodeError, data
 }
 
 type TransferLeaderRequest struct {
-	ShardID           uint64 `json:"shardID"`
+	ClusterName       string `json:"clusterName"`
+	ShardID           uint32 `json:"shardID"`
+	OldLeaderNodeName string `json:"OldLeaderNodeName"`
 	NewLeaderNodeName string `json:"newLeaderNodeName"`
```

Comment:
No need to expose.

Reply:
Only exposed variables could be marshal by json library.